prog/vm/github_runner.rb

```diff
@@ -223,6 +223,33 @@ def setup_info
       UBICLOUD_CACHE_URL=#{Config.base_url}/runtime/github/" | sudo tee -a /etc/environment
     COMMAND
 
+    if vm.vm_host.id == Vm[Config.docker_mirror_server_vm_id]&.vm_host&.id
+      mirror_server_vm = Vm[Config.docker_mirror_server_vm_id]
```

Comment:
small nit: 3 queries to single query

```suggestion
    if (mirror_vm = Vm[Config.docker_mirror_server_vm_id]) && vm.vm_host_id == mirror_vm.vm_host_id
```